src/core_plugins/timelion/server/series_functions/label.js

```diff
@@ -29,15 +30,22 @@ export default new Chainable('label', {
     {
       name: 'label',
       types: ['string'],
-      help: 'Legend value for series. You can use $1, $2, etc, in the string to match up with the regex capture groups'
+      help: i18n.translate('timelion.help.functions.label.args.labelHelpText', {
+        defaultMessage:
+          'Legend value for series. You can use $1, $2, etc, in the string to match up with the regex capture groups',
+      }),
```

Comment:
Add description: `"$1" and "$2" are part of the expression and must not be translated.`